plugins/org.jkiss.dbeaver.ext.altibase.ui/src/org/jkiss/dbeaver/ext/altibase/ui/editors/AltibaseSessionEditor.java

```diff
@@ -0,0 +1,109 @@
+
+import org.eclipse.jface.action.Action;
+import org.eclipse.jface.action.IContributionManager;
+import org.eclipse.jface.action.Separator;
+import org.eclipse.jface.dialogs.IDialogConstants;
+import org.eclipse.jface.dialogs.MessageDialog;
+import org.eclipse.osgi.util.NLS;
+import org.eclipse.swt.widgets.Composite;
+import org.jkiss.dbeaver.ext.altibase.model.AltibaseDataSource;
+import org.jkiss.dbeaver.ext.altibase.model.session.AltibaseServerSession;
+import org.jkiss.dbeaver.ext.altibase.model.session.AltibaseServerSessionManager;
+import org.jkiss.dbeaver.ext.altibase.ui.internal.AltibaseUIMessages;
+import org.jkiss.dbeaver.model.admin.sessions.DBAServerSession;
+import org.jkiss.dbeaver.model.admin.sessions.DBAServerSessionManager;
+import org.jkiss.dbeaver.model.exec.DBCExecutionContext;
+import org.jkiss.dbeaver.ui.DBeaverIcons;
+import org.jkiss.dbeaver.ui.UIIcon;
+import org.jkiss.dbeaver.ui.dialogs.ConfirmationDialog;
+import org.jkiss.dbeaver.ui.views.session.AbstractSessionEditor;
+import org.jkiss.dbeaver.ui.views.session.SessionManagerViewer;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * AltibaseSessionEditor
+ */
+public class AltibaseSessionEditor extends AbstractSessionEditor {
+    private DisconnectSessionAction disconnectSessionAction;
+
+    public AltibaseSessionEditor() {
+    }
+
+    @Override
+    public void createEditorControl(Composite parent) {
+        disconnectSessionAction = new DisconnectSessionAction();
+        super.createEditorControl(parent);
+    }
+
+    @Override
+    protected SessionManagerViewer createSessionViewer(DBCExecutionContext executionContext, Composite parent) {
+        return new SessionManagerViewer<AltibaseServerSession>(this, parent, 
+                new AltibaseServerSessionManager((AltibaseDataSource) executionContext.getDataSource())) {
+
+            @Override
+            protected void contributeToToolbar(DBAServerSessionManager sessionManager, IContributionManager contributionManager) {
+                contributionManager.add(disconnectSessionAction);
+                contributionManager.add(new Separator());
+            }
+            
+            @Override
+            protected void onSessionSelect(DBAServerSession session) {
+                super.onSessionSelect(session);
+                disconnectSessionAction.setEnabled(session != null);
+            }
+        };
+    }
+
+    private class DisconnectSessionAction extends Action {
+        DisconnectSessionAction() {
+            super(AltibaseUIMessages.editors_altibase_session_editor_title_disconnect_session, 
+                 DBeaverIcons.getImageDescriptor(UIIcon.SQL_DISCONNECT));
+        }
+
+        @Override
+        public void run() {
+            final List<DBAServerSession> sessions = getSessionsViewer().getSelectedSessions();
+            final String action = AltibaseUIMessages.editors_altibase_session_editor_action_disconnect 

```

Comment:
It looks like it can be collapsed into one:
`AltibaseUIMessages.editors_altibase_session_editor_action_disconnect_session = Disconnect session`
I do not see other usages of these two variables.

Reply:
Okay, I'll do it.